src/lib/tvgFill.cpp

```diff
@@ -61,6 +61,13 @@ Result Fill::colorStops(const ColorStop* colorStops, uint32_t cnt) noexcept
     pImpl->cnt = cnt;
     memcpy(pImpl->colorStops, colorStops, cnt * sizeof(ColorStop));
 
+    if (pImpl->colorStops[0].offset < 0) pImpl->colorStops[0].offset = 0;
```

Comment:
I'd like to suggest to add a comment here for this exception handling.
example:
//Correct offset range .... xxx